### ISSUE TITLE
rocks: "tarantoolctl rocks install" incorrectly unpack binary packed rocks (luarocks 2)

### ISSUE THREAD

**Issue by olegrok** [tarantool/tarantool#4704]
Tarantool version: 1.10/2.1 (luarocks 2) - not affected 2.2+ (luarocks 3)

OS version: docker image tarantool/tarantool:2.1.2

```sh
> apk add --no-cache git gcc musl-dev make cmake xz unzip
> tarantoolctl rocks install cartridge 1.2.0
> ls -al .rocks/share/tarantool/
total 204
drwxr-xr-x    9 root     root          4096 Dec 25 14:18 .
drwxr-xr-x    3 root     root          4096 Dec 25 14:17 ..
drwxr-xr-x    6 root     root          4096 Dec 25 14:18 cartridge_1_2_0_1-cartridge
-rw-r--r--    1 root     root         21464 Dec 25 14:18 cartridge_1_2_0_1-cartridge.lua
-rw-r--r--    1 root     root          7319 Dec 25 14:17 checks.lua
drwxr-xr-x    2 root     root          4096 Dec 25 14:17 errors
-rw-r--r--    1 root     root         13065 Dec 25 14:17 errors.lua
drwxr-xr-x    2 root     root          4096 Dec 25 14:18 frontend-core
-rw-r--r--    1 root     root          3039 Dec 25 14:18 frontend-core.lua
drwxr-xr-x    2 root     root          4096 Dec 25 14:17 http
-rw-r--r--    1 root     root         89820 Dec 25 14:17 lulpeg.lua
drwxr-xr-x    2 root     root          4096 Dec 25 14:18 membership
-rw-r--r--    1 root     root         27524 Dec 25 14:18 membership.lua
drwxr-xr-x   10 root     root          4096 Dec 25 14:18 rocks
drwxr-xr-x    4 root     root          4096 Dec 25 14:18 vshard
```

Should be `cartridge` instead of `cartridge_1_2_0_1-cartridge`

Then:
```lua
tarantool> require('cartridge')
---
- error: "module 'cartridge' not found:\n\tno field package.preload['cartridge']\n\tno
    file './cartridge.lua'\n\tno file './cartridge/init.lua'\n\tno file './cartridge.so'\n\tno
    file '/opt/tarantool/test/.rocks/share/tarantool/cartridge.lua'\n\tno file '/opt/tarantool/test/.rocks/share/tarantool/cartridge/init.lua'\n\tno
    file '/opt/tarantool/.rocks/share/tarantool/cartridge.lua'\n\tno file '/opt/tarantool/.rocks/share/tarantool/cartridge/init.lua'\n\tno
...
```

Why it happens? Because we use binary packed rocks:
`Installing http://rocks.tarantool.org/cartridge-1.2.0-1.all.rock`

Try to install e.g. vshard with specified version (not packed to rock):
```sh
> tarantoolctl rocks install vshard 0.1.14
Installing http://rocks.tarantool.org/vshard-0.1.14-1.rockspec
> ls .rocks/share/tarantool/
rocks   vshard
```

No any prefixes/postfixes. We are able to require('vshard')

**Comment by rosik** [comment]
Related to https://github.com/tarantool/tarantool/issues/4052

**Comment by kyukhin** [comment]
Fixed by f0b4fd998dd57b3afbeef80eada456d0628de25a
